### PR TITLE
chore: fix lerna publishing steps

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR fixes how lerna operates, so it does what we actually want it to do.

### Context of Change

We no longer have to publish every single package when they haven't changed.

### Type of Change

- [x] Documentation Updates

### CODE REVIEW

**Review by mukulljangid — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -149,11 +149,15 @@ ___
 ___
 NOW YOU HAVE PUBLISHED! But you're not done; we have to notify people!
 ___
-15. On github, click the "releases" link on the right-hand side of the page.
-16. Click "Draft a new release"
-17. Click "Choose a tag" then type in the name of a repo `@` it's newest version to create a new tag (Ex. xrpl@2.0.3)
-18. Edit the name of the release to match the tag (IE \<package\>@\<version\>) and edit the description as you see fit.
-19. Send an email to [xrpl-announce](https://groups.google.com/g/xrpl-announce).
+15. Pull the most recent changes to main locally.
+16. Run `git tag ${tag} -m ${tag}`, where `tag` is the new package and version (e.g. `xrpl@2.1.1`), for each version released.
```
Comment (JST5000) [inline CONTRIBUTING.md:153]: I think this line is a bit ambiguous - Which would be the correct statement?
1. `git xrpl@2.1.1 ${tag} -m ${tag}`
2. `git tag ${xrpl@2.1.1} -m ${xrpl@2.1.1}
3. `git tag xrpl@2.1.1 -m xrpl@2.1.1

Comment (mukulljangid) [reply CONTRIBUTING.md:153]: the 3 rd one is correct

Comment (JST5000) [reply CONTRIBUTING.md:153]:
```suggestion
16. Run `git tag <tagname> -m <tagname>`, where `<tagname>` is the new package and version (e.g. `xrpl@2.1.1`), for each version released.
```

**Review by JST5000 — APPROVED** [review]
LGTM!

**Pull request merged**